src/test/kotlin/stringaddcalculator/StringSeparatorTest.kt

```diff
@@ -1,7 +1,6 @@
 package stringaddcalculator
```

Comment:
테스트 클래스는 테스트 대상 클래스와 같은 패키지에 위치하면 어떨까요?

```suggestion
package stringaddcalculator.domain
```